flink-end-to-end-tests/test-scripts/common.sh

```diff
@@ -587,7 +587,7 @@ function kill_random_taskmanager {
 function setup_flink_slf4j_metric_reporter() {
   INTERVAL="${1:-1 SECONDS}"
   add_optional_lib "metrics-slf4j"
```

Comment:
```suggestion
  add_optional_plugin "metrics-slf4j"
```

This is just a small formality to have the reporter be really loaded as a plugin.

Reply:
yes , you are right , i resumit after change it